### ISSUE TITLE
/availability: Optionally also include pick-up and drop-off information in the response

### ISSUE THREAD

**Issue by nighthawk** [finnishtransportagency/lippu-api#7]
It would be good if the response to `availability` could also include information about the pick-up and drop-off. In particular:

- `pickUpTime`: Estimated time of pick-up. Primarily useful when a pick-up for "now" was requested, but it takes a few minutes for the next vehicle to pick up the passenger.
- `dropOffTime`: Approximate time to the destination. Say, response could get you to the destination quicker but at a higher price, while one gets you there slower but cheaper.
- `pickUpLocation`: In case that the closest pick-up is a bit off a walk away from the requested `fromLat` and `fromLng`
- `dropOffLocation`: Similar but for `toLat` and `toLng`.

FYI, we use a similar structure in our recommendations for APIs for taxi-like services: http://petstore.swagger.io/?url=https://raw.githubusercontent.com/skedgo/TSP-APIs/master/taxi.swagger.yaml#/default/post_quote

**Comment by markkuko** [comment]
Thank you for submitting this issue. We agree, that for demand based transport these are relevant information but we decided that we wont add it now to the API to keep the API compact. We might add this in the future if there is more need for the attributes in the availability response.

**Comment by nighthawk** [comment]
Makes sense. Might be good to clarify in the API docs which transport modes or use cases the API specs are focussed on.
